sdk/build.gradle

```diff
@@ -6,7 +6,7 @@ allprojects {
     apply plugin: 'jacoco'
 
     jacoco {
-        toolVersion '0.7.9'
+        toolVersion '0.7.4.201502262128'
```

Comment:
Why are you using exactly this version?

Reply:
Because it does not work anymore otherwise "Unable to read execution file" which is a binary version problem for reports.

I then executed ./gradlew androidDependencies to find out which version Andoid is using for coverage debug on the real device and it was that version.

Reply:
If this fixes the issue, we can downgrade to the lower version. We can take it separately and fix it later.